inductiva/utils/visualization.py

```diff
@@ -512,3 +512,63 @@ def create_color_plot_movie(
         create_movie_from_frames(frames_dir=tmp_dir,
                                  movie_path=movie_path,
                                  fps=movie_fps)
+
+
+@optional_deps.needs_structures_extra_deps
```

Comment:
With this created just go to the setup.cfg to add the structures_extra package with the extra pyvista.

Reply:
like this, right?

```
needs_structures_extra_deps = functools.partial(
    _needs_optional_deps,
    ["pyvista"],
    _missing_deps_msg("structures"),
)
```